src/components/flags/VariantForm.tsx

```diff
@@ -44,6 +48,9 @@ export default function VariantForm(props: VariantFormProps) {
         handleSubmit(values)
           .then(() => {
             clearError();
+            setSuccess(
+              `Successfully ${submitPhrase.toLocaleLowerCase()}d variant.`
```

Comment:
in some cases we end this message with a `.` and others we don't, like: https://github.com/flipt-io/flipt-ui/pull/52/files#diff-6fdc396f20dbeed901b28539b86f9ee4c7097a9f8aae89882a0c374dddfbb099R51

We should probably be consistent in the messaging/punctation, I'm just not sure which one I prefer (with the period or without). Thoughts?

Reply:
I think I actually like without the `.`

Reply:
Removed the `.` from success messages